.github/workflows/docker.yaml

```diff
@@ -111,14 +111,14 @@ jobs:
       env:
         GITHUB_SHA: ${{ github.sha }}
       run: |
-        echo "DOCKER_TAG=${GITHUB_SHA::7}" >> $GITHUB_ENV
+        DOCKER_TAG=${GITHUB_SHA::7}
```

Comment:
Interesting, they improved that a bit.

Reply:
lol... that's my small change to fix my other workflow.  This wont affect anything :)